examples/simple/readme.md

```diff
@@ -0,0 +1,13 @@
+# Simple
+
+This is an example of the simplest possible configuration. It's great if only have a small number
+of Windows APIs that you'd like to use. For anything beyond that, you'll want to create a bindings
```

Comment:
Should we link to a readme for a bindings example in case folks are unfamiliar with the bindings pattern?